src/components/outside/home/categories/CategorieCard.js

```diff
@@ -1,14 +1,18 @@
 import React from "react";
 import PropTypes from "prop-types";
 import { CardWrapper } from "./CategorieCard.styled";
+import { Link, Outlet } from "react-router-dom";
 
 export const CategorieCard = ({ name, main_image, renderImage = true, id }) => {
   const { url, alt } = main_image;
-
   return (
     <CardWrapper id={id}>
-      <div className="CardWrapper-title">{name}</div>
+      <div className="CardWrapper-title">
+        {renderImage && <Link to={`category=${id}`}>{name}</Link>}
```

Comment:
another way to do this is 
```suggestion
        {
         renderImage ? 
           <Link to={`category=${id}`}>{name}</Link>
         : 
           name
        }
```
That way you avoid having the two conditionals